assets/app/scripts/controllers/pods.js

```diff
@@ -12,38 +12,14 @@ angular.module('openshiftConsole')
     $scope.pods = {};
     $scope.podsByLabel = {};
 
-    var callback = function(action, pod) {
+    var callback = function(pods, action, pod) {
```

Comment:
should we remove the optional args if we're not using them in a particular callback?